### DISCUSSION TITLE
Change the Dashboard and Netbox web logo

### DISCUSSION THREAD

**Discussion by andersonmavi30** [netbox-community/netbox#9467] [Q&A]
Hello everyone, I am interested in changing the Dashboard logo and changing the admin or web logo logo to log in, already try to look at/opt/netbox/but I can not find the info where the image must be replaced And manage to customize it, also look for Google and here in the discussions but there is nothing concrete, someone could please help me, thank you.

**Comment by bogdancordos** [comment]
Hi, as per documentation found at https://docs.netbox.dev/en/stable/development/style-guide/ , the logo file is at /static/docs/netbox_logo.svg
You might need to update this every time you upgrade your NetBox as the file will be overwritten.

**Comment by adreiignat** [comment]
Could you help me with the same problem ? i have installed netbox with docker and the file netbox_logo.svg is under many folders :
/var/lib/docker/overlay2/4c2d9ad6a4d4b7e0a6e55831a936dcf4995e66f691568fa68b24b4bbb761567b/merged/opt/netbox/netbox/project-static/img/netbox_logo.svg
/var/lib/docker/overlay2/4c2d9ad6a4d4b7e0a6e55831a936dcf4995e66f691568fa68b24b4bbb761567b/merged/opt/netbox/netbox/project-static/docs/netbox_logo.svg
/var/lib/docker/overlay2/4c2d9ad6a4d4b7e0a6e55831a936dcf4995e66f691568fa68b24b4bbb761567b/merged/opt/netbox/netbox/static/docs/netbox_logo.svg
/var/lib/docker/overlay2/4c2d9ad6a4d4b7e0a6e55831a936dcf4995e66f691568fa68b24b4bbb761567b/merged/opt/netbox/netbox/static/netbox_logo.svg
/var/lib/docker/overlay2/4c2d9ad6a4d4b7e0a6e55831a936dcf4995e66f691568fa68b24b4bbb761567b/merged/opt/netbox/docs/netbox_logo.svg
/var/lib/docker/overlay2/3fd47029b564d30261c6e8d62d8d3334c06ab900b7ef666a0d2eb9ae199bc148/diff/opt/netbox/netbox/project-static/img/netbox_logo.svg
/var/lib/docker/overlay2/3fd47029b564d30261c6e8d62d8d3334c06ab900b7ef666a0d2eb9ae199bc148/diff/opt/netbox/docs/netbox_logo.svg
/var/lib/docker/overlay2/a82240f77c60534ca9f96658feb98f65a9b04f218cdc2ecdf569c4cc71ae8203/merged/opt/netbox/netbox/project-static/img/netbox_logo.svg
/var/lib/docker/overlay2/a82240f77c60534ca9f96658feb98f65a9b04f218cdc2ecdf569c4cc71ae8203/merged/opt/netbox/netbox/project-static/docs/netbox_logo.svg
/var/lib/docker/overlay2/a82240f77c60534ca9f96658feb98f65a9b04f218cdc2ecdf569c4cc71ae8203/merged/opt/netbox/netbox/static/docs/netbox_logo.svg
/var/lib/docker/overlay2/a82240f77c60534ca9f96658feb98f65a9b04f218cdc2ecdf569c4cc71ae8203/merged/opt/netbox/netbox/static/netbox_logo.svg
/var/lib/docker/overlay2/a82240f77c60534ca9f96658feb98f65a9b04f218cdc2ecdf569c4cc71ae8203/merged/opt/netbox/docs/netbox_logo.svg
/var/lib/docker/overlay2/6a4319641921ac9af07a04b18cf905681887de63fc5131c7bdc2636fc3f31e07/merged/opt/netbox/netbox/project-static/img/netbox_logo.svg
/var/lib/docker/overlay2/6a4319641921ac9af07a04b18cf905681887de63fc5131c7bdc2636fc3f31e07/merged/opt/netbox/netbox/project-static/docs/netbox_logo.svg
/var/lib/docker/overlay2/6a4319641921ac9af07a04b18cf905681887de63fc5131c7bdc2636fc3f31e07/merged/opt/netbox/netbox/static/docs/netbox_logo.svg
/var/lib/docker/overlay2/6a4319641921ac9af07a04b18cf905681887de63fc5131c7bdc2636fc3f31e07/merged/opt/netbox/netbox/static/netbox_logo.svg
/var/lib/docker/overlay2/6a4319641921ac9af07a04b18cf905681887de63fc5131c7bdc2636fc3f31e07/merged/opt/netbox/docs/netbox_logo.svg
/var/lib/docker/overlay2/38d2443b02c82a10ac5c52a5066d83e55f61f264a7618875bc92f68a6263c85f/diff/opt/netbox/netbox/static/docs/netbox_logo.svg
/var/lib/docker/overlay2/38d2443b02c82a10ac5c52a5066d83e55f61f264a7618875bc92f68a6263c85f/diff/opt/netbox/netbox/static/netbox_logo.svg
/var/lib/docker/overlay2/38d2443b02c82a10ac5c52a5066d83e55f61f264a7618875bc92f68a6263c85f/diff/opt/netbox/netbox/project-static/docs/netbox_logo.svg

:)

**Reply by lowinger42** [reply]
I'm using Apache to front netbox/docker. It is easy to catch the request for the logo file and redirect it. No changes need to be done in Netbox. I guess you could do the same with nginx, caddy, traefic etc.

/etc/apache2/sites-enabled/netbox.conf

<VirtualHost <ip address>:443>
    ServerName netbox.example.com
    ServerAdmin support@example.com

    Alias "/static/netbox_logo.svg" "/opt/netbox/logo/netbox_logo.svg"
    <Directory "/opt/netbox/logo">
        Require all granted
    </Directory>

    ProxyPass /static/netbox_logo.svg !
    ProxyPass "/" "http://172.27.17.91:8000/"
    ProxyPassReverse "/" "http://172.27.17.91:8000/"
    ProxyPreserveHost On
    RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}

    <Proxy *>
        Order deny,allow
        Allow from all
        Allow from localhost
    </Proxy>

    SSLEngine on
    SSLCertificateFile /etc/ssl/nitor/netbox.itn.nu.crt
    SSLCertificateKeyFile /etc/ssl/nitor/netbox.itn.nu.key

    ErrorLog ${APACHE_LOG_DIR}/netbox-error.log
    CustomLog ${APACHE_LOG_DIR}/netbox-access.log combined
</VirtualHost>

**Reply by candlerb** [reply]
If you're using docker, you can [extend the image](https://github.com/netbox-community/netbox-docker/wiki/Build#extend-the-image) with your own customizations. Or you could just volume-mount a single svg file which overrides the one of interest.

**Reply by adreiignat** [reply]
@candlerb thx for the answer.
my question was from the list of the folders listed by me, there is the same file with the same name and i don't know witch file to replace with my logo file.

**Reply by candlerb** [reply]
None of them - you don't touch any of the docker overlays.

Right-click/inspect the image in your browser you'll see it's `/static/netbox_logo.svg` in your browser. That's served from `/opt/netbox/netbox/static/netbox_logo.svg`.

However the `collectstatic` task pulls it from somewhere else - I think `/opt/netbox/netbox/project-static/img/netbox_logo.svg`. If the docker container runs that task at startup then you'll have to override the latter file instead.

**Comment by tacerus** [comment]
For v4 and bright/dark mode, mount to these paths instead:

```
/opt/netbox/netbox/project-static/img/logo_netbox_bright_teal.svg
/opt/netbox/netbox/static/logo_netbox_bright_teal.svg
/opt/netbox/netbox/project-static/img/logo_netbox_dark_teal.svg
/opt/netbox/netbox/static/logo_netbox_dark_teal.svg
```
